README.md

```diff
@@ -1,10 +1,45 @@
-
-TextAnalysis
-============
+# TextAnalysis
 
 A Julia package for working with text.
 
 [![Travis](https://travis-ci.org/JuliaText/TextAnalysis.jl.svg?branch=master)](https://travis-ci.org/JuliaText/TextAnalysis.jl)
 [![Appveyor](https://ci.appveyor.com/api/projects/status/aviks/textanalysis-jl?svg=true)](https://ci.appveyor.com/project/aviks/textanalysis-jl)
-<!-- [![](https://img.shields.io/badge/docs-stable-blue.svg)](https://juliatext.github.io/TextAnalysis.jl/stable) -->
+<!--[![](https://img.shields.io/badge/docs-stable-blue.svg)](https://juliatext.github.io/TextAnalysis.jl/stable) -->
 [![](https://img.shields.io/badge/docs-dev-blue.svg)](https://juliatext.github.io/TextAnalysis.jl/dev)
+[![](https://img.shields.io/badge/docs-latest-blue.svg)](https://juliatext.github.io/TextAnalysis.jl/latest)
+
+Please see the detailed `manual` and `API Documentation` for the linked above. The package also comes with a set of `docstrings` for offline documentation access.
+
+## Introduction
+
+
+## Installation
+
+```julia
+pkg> add TextAnalysis
+```
+
+You may want to work on the master branch for the latest features (as mentioned in NEWS.md).
+
+```julia
+pkg> add TextAnalysis#master
+```
+
+## Features
+
+Refer to NEWS.md for the latest features.
```

Comment:
Maybe say something like "TextAnalysis provides support for standard NLP tools and models in the Julia language. Please see the [documentation](http://......) for more. "

Reply:
Done.